.circleci/config.yml

```diff
@@ -18,7 +18,9 @@ jobs:
     steps:
       - checkout
       - run: bundle install
-      - run: bundle exec rake
+      - run:
+          name: Run Minitest
+          command: bundle exec rake test
```

Comment:
This change is unnecessary because tests and static analysis and others are performed by default rake task.
https://github.com/rubocop-hq/rubocop-minitest/blob/e239e39e5a2f42b8748a794e4cd2294a5571445e/Rakefile#L32-L37

Reply:
Oh. I didn't see that. My bad.